app/lib/common/widgets/spaces/space_with_profile_card.dart

```diff
@@ -139,19 +138,4 @@ class SpaceWithProfileCard extends StatelessWidget {
     );
   }
 
-  ShapeBorder? renderShape(BuildContext context) {
```

Comment:
So... You just removed the entire feature but left the options (`shape` and `withBorder`) on the widget? Why? Either the widget supports those features or the options should be removed as well, which would also ensure that wherever that might be used is properly updated to the new feature set...